### PR TITLE
minimal check for vpn

### PR DESCRIPTION
**Pull request by boshek**

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -72,6 +75,7 @@ set_network_path <- function(x) {
 # (path beginning with \\). Please use the mapped drive letter (e.g., P:/etc).",
 #          call. = FALSE)
 # }
+  check_if_available()
```
Comment (ateucher) [inline R/utils.R]: Won't this always fail if someone hasn't set it yet? Not sure if needs to be in this function at all actually; someone might want to set it while not connected...

File: R/utils.R [hunk 2]
```diff
@@ -104,6 +108,7 @@ set_network_path <- function(x) {
 #' get_network_path()
 #' }
 get_network_path <- function() {
+  check_if_available()
```
Comment (ateucher) [inline R/utils.R]: Also here - I think if this passes then the rest of the function will always pass?

Comment (ateucher) [reply R/utils.R]: Maybe it should be between lines 115 and 116 - (I.e., call just before returning the path)?

File: R/utils.R [hunk 3]
```diff
@@ -40,6 +40,9 @@ path relative to the network path set with `set_network_path()`.")
   if (!dir.exists(root)) {
     stop(root, " Does not exist. Are you connected to the network?")
   }
```
Comment (ateucher) [inline R/utils.R:42]: I think this is made redundant by `check_if_available()`, especially if it's called inside `get_network_path()`

**Review by ateucher — CHANGES_REQUESTED** [review]
Great idea, I've left some comments about the logic about where it's being called and how it interacts with `get_network_path()`

File: R/utils.R (hunk 2 above)
Comment (boshek) [reply R/utils.R]: Agreed. See changes.

File: R/utils.R (hunk 1 above)
Comment (boshek) [reply R/utils.R]: Added `check_if_set`

**Comment by boshek** [comment]
@ateucher some changes to consider

**Comment by boshek** [comment]
@ateucher some changes to consider.

**Review by ateucher — APPROVED** [review]
Looks excellent! Love the sitrep function!

**Pull request merged**